packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2912,20 +2918,41 @@ Iterable<DiagnosticsNode> _describeRelevantUserCode(Element element) {
       ErrorSpacer(),
     ];
   }
+
+  bool isOverflowError() {
+    for (final DiagnosticsNode node in properties) {
+      if (node is ErrorSummary) {
+        if (node.value.isNotEmpty
+            && node.value.first is String
+            && (node.value.first as String).contains('A RenderFlex overflowed by')) {
```

Comment:
nit: contains --> startsWith

avoid `as` by creating a final variable
final Object first = node.value.first;
if (first is String && fist.contains(...))
will then type check.

Reply:
done